modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ExportImportUtil.java

```diff
@@ -0,0 +1,170 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.web.internal.portlet.action.test;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.model.Layout;
+import com.liferay.portal.kernel.model.LayoutConstants;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.portlet.LiferayPortletRequest;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCActionCommand;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCResourceCommand;
+import com.liferay.portal.kernel.test.ReflectionTestUtil;
+import com.liferay.portal.kernel.test.portlet.MockLiferayPortletActionRequest;
+import com.liferay.portal.kernel.test.portlet.MockLiferayPortletActionResponse;
+import com.liferay.portal.kernel.test.portlet.MockLiferayResourceRequest;
+import com.liferay.portal.kernel.test.portlet.MockLiferayResourceResponse;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.ArrayUtil;
+import com.liferay.portal.kernel.util.File;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.kernel.util.Portal;
+import com.liferay.portal.kernel.util.ProxyUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.kernel.util.Validator;
+import com.liferay.portal.kernel.util.WebKeys;
+import com.liferay.portal.model.impl.LayoutImpl;
+import com.liferay.portal.upload.test.util.UploadTestUtil;
+
+import java.util.Objects;
+
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.mock.web.MockMultipartHttpServletRequest;
+
+/**
+ * @author Guilherme Sá
+ */
+public class ExportImportUtil {
```

Comment:
After talk to Carol, we came to the conclusion that would be better to create a `Base` abstract class instead of the `TestUtil`.

So we can replace the line 47 for: `public abstract class BaseExportImportTestCase`

---

The `ExportImportObjectDefinitionTest` and `ExportImportObjectFolderTest` will extend the `BaseExportImportTestCase`.

We need to rename the classes below:
- `ExportImportObjectDefinitionTest` -> `ObjectDefinitionExportImportTest`
- `ExportImportObjectFolderTest` -> `ObjectFolderExportImportTest`

---

The `BaseExportImportTestCase` must have a `testExportImport()` method that will be responsible for carry all the logic of the export and import.

You will see that the `objectDefinition = _getObjectDefinition(objectDefinitionName);` from [here](https://github.com/liferay-objects/liferay-portal/blob/0c5eb749f4f6f60dccb82ae625e4237b3b681639/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ExportImportObjectDefinitionTest.java#L96) can now be a method that we will override in the `ObjectDefinitionExportImportTest` and `ObjectFolderExportImportTest`.

---


Feel free to ask if you have any questions